graylog2-web-interface/src/routing/AppFacade.jsx

```diff
@@ -10,13 +10,12 @@ const SessionStore = StoreProvider.getStore('Session');
 const ServerAvailabilityStore = StoreProvider.getStore('ServerAvailability');
 const CurrentUserStore = StoreProvider.getStore('CurrentUser');
 
-import 'javascripts/shims/styles/shim.css';
-import 'stylesheets/bootstrap.min.css';
+import 'bootstrap/dist/css/bootstrap.css';
```

Comment:
Shouldn't this be [`bootstrap/dist/css/bootstrap.min.css`](https://github.com/twbs/bootstrap/blob/v3.3.7/dist/css/bootstrap.min.css)?